docs/index.rst

```diff
@@ -83,6 +83,21 @@ On this tab a number of configuration parameters are available;
     | |opcua_2| |
     +-----------+
 
+The Filtering tab allows setting the filtering config to include/exclude the OPCUA nodes to subscribe to.
```

Comment:
The Filtering allows you to include or exclude OPCUA nodes using a regular expression to filter Browse Names.